### PR TITLE
TOOLS-3500 Remove use of the `filemd5` command in test code

### PR DESCRIPTION
**Pull request by autarch**
This usage in the test code seems to have been related to something the `mongofiles` code _used_ to do quite some time in the past. There are no longer any references to the `filemd5` command anywhere else in the tools codebase.

The `filemd5` command is going to be deprecated and eventually removed from the Server.

### CODE REVIEW

**Review by huan-Mongo — CHANGES_REQUESTED** [review]
I think the `filemd5` command was to validate the result of `mongofiles` command, so we cannot remove it.

File: test/legacy42/jstests/tool/gridfs.js [hunk 1]
```diff
@@ -32,12 +29,6 @@ function testGridFS(name) {
...
 
-    // Call filemd5 ourself and check results.
-    var res = d.runCommand({filemd5: fileObj._id});
-    print("filemd5 output: " + tojson(res));
-    assert(res.ok);
-    assert.eq(rawmd5, res.md5);
-
     var numChunks = d.fs.chunks.find({files_id: fileObj._id}).itcount();
     // var numChunks = d.fs.chunks.count({files_id: fileObj._id}) // this is broken for now
     assert.eq(numChunks, res.numChunks);
```
[the remark this answers is not in the document]
Comment (autarch) [reply test/legacy42/jstests/tool/gridfs.js:34]: I replaced this with some code that downlaods the file and compares the downloaded file's md5 hash to the original's.

[the remark this answers is not in the document]
Comment (huan-Mongo) [reply test/legacy42/jstests/tool/gridfs.js:34]: So this line should be removed, right?

**Review by huan-Mongo — COMMENTED** [review]
.

File: test/legacy42/jstests/tool/gridfs.js (hunk 1 above)
[the remark this answers is not in the document]
Comment (tfogo) [reply test/legacy42/jstests/tool/gridfs.js:34]: Yeah it looks like it should be removed. I'm confused why the tests didn't fail. It looks like res should be undefined.

[the remark this answers is not in the document]
Comment (autarch) [reply test/legacy42/jstests/tool/gridfs.js:34]: We don't run any of the "legacy" tests for patches right now. I think that was an oversight from when I moved the GitHub PR aliases to the Evergreen conf and wrote code to generate that list of aliases. I just updated that to include the "legacy" tests.

[the remark this answers is not in the document]
Comment (autarch) [reply test/legacy42/jstests/tool/gridfs.js]: So when I added the legacy tests back to the GH PR aliases, they did fail. I pushed a fix for this test to remove all references to `res`.

[the remark this answers is not in the document]
Comment (autarch) [reply test/legacy42/jstests/tool/gridfs.js]: Ok, the tests are now running in CI and they passed.

**Review by huan-Mongo — APPROVED** [review]
LGTM % the lint failure.

**Comment by autarch** [comment]
> LGTM % the lint failure.

The lint failure is also happening in `master`. It looks like an infra thing:

> [2024/04/15 23:56:42.726] sh: line 5: /opt/node/bin/npm: No such file or directory

https://parsley.mongodb.com/evergreen/mongo_tools_static_lint_js_patch_ac0ae2543783578632e6beefde9914b9cc123eb4_661d4cfa1b2cab000771c4fb_24_04_15_15_51_23/0/task?bookmarks=0,51,74

**Review by tfogo — APPROVED** [review]
LGTM

**Pull request merged**